alloydb-jdbc-connector/src/test/java/com/google/cloud/alloydb/ConnectionInfoCacheTest.java

```diff
@@ -237,209 +184,22 @@ public void testGetConnectionInfo_scheduledNextOperationImmediately_onCertificat
-    List<X509Certificate> certificateChain =
-        Arrays.asList(
-            testCertificates.getIntermediateCertificate(), testCertificates.getRootCertificate());
-    connectionInfoRepo.addResponses(
-        () ->
-            new ConnectionInfo(
-                TEST_INSTANCE_IP,
-                TEST_INSTANCE_ID,
-                testCertificates.getEphemeralCertificate(keyPair.getPublic(), ONE_HOUR_FROM_NOW),
-                certificateChain,
-                testCertificates.getRootCertificate()),
-        () ->
-            new ConnectionInfo(
-                TEST_INSTANCE_IP,
-                TEST_INSTANCE_ID,
-                testCertificates.getEphemeralCertificate(keyPair.getPublic(), TWO_HOURS_FROM_NOW),
-                certificateChain,
-                testCertificates.getRootCertificate()));
-    DefaultConnectionInfoCache connectionInfoCache =
-        new DefaultConnectionInfoCache(
-            MoreExecutors.listeningDecorator(executor),
-            connectionInfoRepo,
-            instanceName,
-            keyPair,
-            new RefreshCalculator(),
-            spyRateLimiter);
-
-    executor.runNextPendingCommand(); // Simulate completion of background thread.
+    try {
+      Thread.sleep(1000);
+    } catch (InterruptedException e) {
+      throw new RuntimeException(e);
+    }
 
-    // Before force refresh, the first refresh data is available.
     ConnectionInfo connectionInfo = connectionInfoCache.getConnectionInfo();
-    assertThat(connectionInfoRepo.getIndex()).isEqualTo(1);
-    assertThat(
-            connectionInfo
-                .getClientCertificate()
-                .getNotAfter()
-                .toInstant()
-                .truncatedTo(ChronoUnit.SECONDS))
-        .isEqualTo(ONE_HOUR_FROM_NOW.truncatedTo(ChronoUnit.SECONDS));
-
-    connectionInfoCache.forceRefresh();
-
-    // Refresh data hasn't changed because we re-use the existing connection info.
-    connectionInfo = connectionInfoCache.getConnectionInfo();
-    assertThat(
-            connectionInfo
-                .getClientCertificate()
-                .getNotAfter()
-                .toInstant()
-                .truncatedTo(ChronoUnit.SECONDS))
-        .isEqualTo(ONE_HOUR_FROM_NOW.truncatedTo(ChronoUnit.SECONDS));
-
-    executor.runUntilIdle(); // Simulate completion of background thread.
-
-    // After the force refresh, new refresh data is available.
-    connectionInfo = connectionInfoCache.getConnectionInfo();
-    assertThat(connectionInfoRepo.getIndex()).isEqualTo(2);
-    assertThat(
-            connectionInfo
-                .getClientCertificate()
-                .getNotAfter()
-                .toInstant()
-                .truncatedTo(ChronoUnit.SECONDS))
-        .isEqualTo(TWO_HOURS_FROM_NOW.truncatedTo(ChronoUnit.SECONDS));
-  }
 
-  @Test
-  public void testForceRefresh_refreshCalledOnlyOnceDuringMultipleCalls()
-      throws InterruptedException {
```

Comment:
This is tested by `RefresherTest.testThatForceRefreshBalksWhenAScheduledRefreshIsInProgress()` and `RefresherTest.testThatForceRefreshBalksWhenAForceRefreshIsInProgress()`